src/blockchain_txn_token_convert_v1.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package helium;
+
+import "blockchain_token_type_v1.proto";
+
+message blockchain_txn_token_convert_v1 {
```

Comment:
```suggestion
message blockchain_txn_token_redeem_v1 {
```

Wondering if `redeem` is better than `convert` here since the action is to redeem a MOBILE (for example) for the appropriate amount of HNT in the treasury.

Reply:
we don't implement this txn in the work so far, so we can change it here or later, I don't have an opinion.

Reply:
Addressed in [49b084f](https://github.com/helium/proto/pull/133/commits/49b084f93a223bac7801f1be606c06d2e9ad2b74)